src/Cake.Common/Tools/InnoSetup/InnoSetupSettings.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Cake.Core.IO;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.InnoSetup
+{
+    /// <summary>
+    /// Contains settings used by the <see cref="InnoSetupRunner"/>.
+    /// </summary>
+    public sealed class InnoSetupSettings : ToolSettings
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InnoSetupSettings"/> class with the default settings.
+        /// </summary>
+        public InnoSetupSettings()
```

Comment:
@patriksvensson do you know why the above would be commented out?  If this is a rule that we want to enforce, best to leave StyleCop to do that work.